lib/checks/aria/braille-label-equivalent-evaluate.js

```diff
@@ -0,0 +1,18 @@
+import { accessibleTextVirtual } from '../../commons/text';
+
+/**
+ * Check that if aria-braillelabel is not empty, the element has an accessible text
+ * @memberof checks
+ * @return {Boolean}
+ */
+export default function brailleLabelEquivalentEvaluate(
+  node,
+  options,
+  virtualNode
+) {
+  const brailleLabel = virtualNode.attr('aria-braillelabel') ?? '';
+  if (!brailleLabel.trim()) {
```

Comment:
Should we use `sanitize` from our text functions here instead (similar to aria-label and aria-labelledby checks)?

Reply:
We probably shouldn't use it for aria-labelledby either. What sanitize does that trim doesn't do is replace duplicate whitespace characters in between words with single space characters. We don't need that here.